lib/peep.rb

```diff
@@ -0,0 +1,45 @@
+require "./lib/select_database"
+require "./lib/database_connection"
+
+class Peep 
+
+  attr_reader :id, :body, :posted_at, :author
+
+  def initialize(id, body, date, user_id)
+    @id = id
+    @body = body
+    @posted_at = date
+    @user_id = user_id
+    @author = find_author
+  end
+
+  def self.all
+    select_database
+
+    peeps = DatabaseConnection.query("SELECT * FROM peeps ORDER BY id DESC")
+
+    peeps.map do |peep|
+      Peep.new(peep["id"], peep["body"], peep["posted_at"], peep["user_id"])
+    end
+  end
+
+  def self.add(peep, user_id)
+    select_database
+
+    result = DatabaseConnection.query(
+                              "INSERT INTO peeps (body, user_id) 
+                              VALUES ('#{peep}', '#{user_id}') 
```

Comment:
In the group code review, we talked about escaping values - injecting values directly into your SQL leaves you open to SQL injection attacks. What is somebody peeped:

```
'); DROP TABLE Peeps;
```